src/main/java/ch/naviqore/raptor/RaptorBuilder.java

```diff
@@ -20,19 +18,30 @@
  *
  * @author munterfi
  */
-@NoArgsConstructor(access = AccessLevel.PACKAGE)
 @Log4j2
 public class RaptorBuilder {
 
+    private final static int DEFAULT_SAME_STATION_TRANSFER_TIME = 120;
```

Comment:
Maybe remove this? I would not provide default values. 

Reason: This scatters default values all over the project and it will be difficult to track which default values are set and where.

Always require to pass a value for the same station transfer time, which we can fetch from the application.properties.

Reply:
cc10db6 there is more passing down than expected when I started implementing, but still probably better this way.